includes/apple-exporter/class-html.php

```diff
@@ -70,20 +70,12 @@ public function format( $html ) {
 		// Since wp_kses has an issue with some <script> tags, proactively strip them.
 		$html = preg_replace( '/<script[^>]*?>.*?<\/script>/', '', $html );
 
+		// Remove any tempty tags.
```

Comment:
Let's move this back below line 77 so that if `wp_kses` produces empty tags, we are then removing them.